### DISCUSSION TITLE
Is there any way to add type definitions to Json?

### DISCUSSION THREAD

**Discussion by yepMad** [prisma/prisma#11230] [Q&A] [answered]
Hi! I use MongoDB and I have fields with definition like "Json". Is it possible to define a type declaration?

![image](https://user-images.githubusercontent.com/22564368/149848771-aec357c5-5cbd-4aa2-bdfb-7fe5ff0bd445.png)

**Comment by janpio** [comment, marked as answer]
Not yet, you should follow https://github.com/prisma/prisma/issues/6708 though which is a feature request for this.

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that have already been marked as answered but remain open.

If this discussion still requires further input or clarification, feel free to reopen it or start a new one with updated details. Your contributions are invaluable to the community, and we’re here to help!

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and ongoing support of the Prisma community!
